src/Carbon/Carbon.php

```diff
@@ -288,6 +288,12 @@ public function __construct($time = null, $tz = null)
             $time = $testInstance->toDateTimeString();
         }
 
+        if (is_null($time) || $time === 'now') {
+            $dateTime = new \DateTime('now', static::safeCreateDateTimeZone($tz));
+            $micro = preg_replace('/0\.(\d{6}).*/', '$1', microtime());
+            $time = $dateTime->format(static::DEFAULT_TO_STRING_FORMAT).'.'.$micro;
+        }
+
         parent::__construct($time, static::safeCreateDateTimeZone($tz));
```

Comment:
`static::safeCreateDateTimeZone($tz)` is calculated twice, this can be extracted into a variable